tests/test_tools_manually.yml

```diff
@@ -123,3 +123,24 @@
       md5sum: c09dff3f145d77d4848992e244811c08
     - path: results/variant_calling/mutect2/sample4_vs_sample3/sample4_vs_sample3.mutect2.vcf.gz.tbi
     # binary changes md5sums on reruns
+- name: Run full pipeline on tumoronly with most tools
+  command: nextflow run . -profile test --input tests/csv/3.0/fastq_tumor_only.csv --tools cnvkit,freebayes,merge,mpileup,mutect2,snpeff,strelka,tiddit,vep --outdir results
```

Comment:
does snpeff and vep work here? because of 👨 vs 🪱 I mean?

Reply:
yes

Reply:
this is profile test, so the same data we use for the annotation test.
But to confirm, I tried the command again, and this works

Reply:
hmmm I could have sworn we ran into issues before, that multiqc didn't run when combining step variantcalling with snpeff. anyways, good that it works!

Reply:
🤦‍♀️ Susanne fixed this mqc iirc and with the new release it should work now

Reply:
Oh yeah, I remember now